Makefile

```diff
@@ -1923,6 +1829,24 @@ uid.o: \
 compile uid.c uidgid.h subfd.h substdio.h exit.h
 	./compile uid.c
 
+uids.c: \
+auto-str conf-users
+	./auto-str auto_usera `head -n 1 conf-users` > uids.c.tmp
```

Comment:
Possibly goofy idea, but for visual consistency, maybe pipe this (and `auto_groupq`) through `tail -n -1` also?

Reply:
I have added some spaces for alignment.